From 455441d55fe88846767ddb3e1d380f1e65788bad Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Wed, 29 Jul 2020 19:56:12 +0200
Subject: [PATCH 2/3] Set PLUGIN_CLASS_NAME in .pro files

This resolves the erronous generation of Qt5Contacts_.cmake,
Qt5Organizer_.cmake & Qt5Versit_.cmake by setting proper names to the
plugins.

Change-Id: I094ce93b53fc2e56b45bf5edf05d6da17ed435e3
---
 src/plugins/contacts/memory/memory.pro                           | 1 +
 .../contacts/serviceactionmanager/serviceactionmanager.pro       | 1 +
 src/plugins/organizer/memory/memory.pro                          | 1 +
 src/plugins/organizer/skeleton/skeleton.pro                      | 1 +
 src/plugins/versit/backuphandler/backuphandler.pro               | 1 +
 src/plugins/versit/vcardpreserver/vcardpreserver.pro             | 1 +
 6 files changed, 6 insertions(+)

diff --git a/src/plugins/contacts/memory/memory.pro b/src/plugins/contacts/memory/memory.pro
index 96a1f49f..455b4a5d 100644
--- a/src/plugins/contacts/memory/memory.pro
+++ b/src/plugins/contacts/memory/memory.pro
@@ -2,6 +2,7 @@ TARGET = qtcontacts_memory
 QT = core contacts
 
 PLUGIN_TYPE = contacts
+PLUGIN_CLASS_NAME = QMemoryContactsPlugin
 load(qt_plugin)
 
 HEADERS += \
diff --git a/src/plugins/contacts/serviceactionmanager/serviceactionmanager.pro b/src/plugins/contacts/serviceactionmanager/serviceactionmanager.pro
index 3fcd8237..a1fde021 100644
--- a/src/plugins/contacts/serviceactionmanager/serviceactionmanager.pro
+++ b/src/plugins/contacts/serviceactionmanager/serviceactionmanager.pro
@@ -2,6 +2,7 @@ TARGET = qtcontacts_serviceactionmanager
 QT = core contacts
 
 PLUGIN_TYPE = contacts
+PLUGIN_CLASS_NAME = QServiceActionManagerContactsPlugin
 load(qt_plugin)
 
 HEADERS += \
diff --git a/src/plugins/organizer/memory/memory.pro b/src/plugins/organizer/memory/memory.pro
index 599a5b76..f98890f8 100644
--- a/src/plugins/organizer/memory/memory.pro
+++ b/src/plugins/organizer/memory/memory.pro
@@ -2,6 +2,7 @@ TARGET = qtorganizer_memory
 QT = core organizer-private
 
 PLUGIN_TYPE = organizer
+PLUGIN_CLASS_NAME = QMemoryOrganizerPlugin
 load(qt_plugin)
 
 HEADERS += \
diff --git a/src/plugins/organizer/skeleton/skeleton.pro b/src/plugins/organizer/skeleton/skeleton.pro
index 3cf33a9a..6eac4e4f 100644
--- a/src/plugins/organizer/skeleton/skeleton.pro
+++ b/src/plugins/organizer/skeleton/skeleton.pro
@@ -2,6 +2,7 @@ TARGET = qtorganizer_skeleton
 QT += organizer
 
 PLUGIN_TYPE = organizer
+PLUGIN_CLASS_NAME = QSkeletonOrganizerPlugin
 load(qt_plugin)
 
 HEADERS += qorganizerskeleton_p.h
diff --git a/src/plugins/versit/backuphandler/backuphandler.pro b/src/plugins/versit/backuphandler/backuphandler.pro
index d5179407..330448d0 100644
--- a/src/plugins/versit/backuphandler/backuphandler.pro
+++ b/src/plugins/versit/backuphandler/backuphandler.pro
@@ -2,6 +2,7 @@ TARGET = qtversit_backuphandler
 QT += contacts versit-private
 
 PLUGIN_TYPE = versit
+PLUGIN_CLASS_NAME = QBackupHandlerVersitPlugin
 load(qt_plugin)
 
 HEADERS += backupvcardhandler.h
diff --git a/src/plugins/versit/vcardpreserver/vcardpreserver.pro b/src/plugins/versit/vcardpreserver/vcardpreserver.pro
index e087d84e..d3898536 100644
--- a/src/plugins/versit/vcardpreserver/vcardpreserver.pro
+++ b/src/plugins/versit/vcardpreserver/vcardpreserver.pro
@@ -2,6 +2,7 @@ TARGET = qtversit_vcardpreserver
 QT += contacts versit
 
 PLUGIN_TYPE = versit
+PLUGIN_CLASS_NAME = QVCardPreserverVersitPlugin
 load(qt_plugin)
 
 HEADERS += vcardpreserver.h
-- 
2.27.0

